source/resources/links/feed.xml.builder

```diff
@@ -21,7 +21,7 @@ xml.feed "xmlns" => "http://www.w3.org/2005/Atom" do
     DOC
     xml.entry do
       xml.title link.title
-      xml.link "rel" => "alternate", "href" => link.url
+      xml.link "rel" => "alternate", "href" => tracking_url
       xml.id link.url
```

Comment:
This is staying as-is because it's the primary identifier for the entry, and it makes sense to leave it as the real URL